packages/main/src/features/InputElementsFormSupport.js

```diff
@@ -26,6 +26,41 @@ class FormSupport {
 		}
 	}
 
+	static syncNativeVisibleInput(element, nativeInputUpdateCallback, nativeInputChangeCallback) {
```

Comment:
This looks too custom-tailored for the type="file" input, so I'd suggest we call it syncNativeFileInput.